README.md

```diff
@@ -0,0 +1,78 @@
+# Python Automated Testing Example
```

Comment:
You don't need this README as part of this commit - you can remove this or modify it to fit your current project structure.

Reply:
okay i'll fix that